qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -174,15 +175,32 @@ def __init__(
         self._register_parameter(self.meas_freq, ())
 
         # Backends with a single qubit may not have a coupling map.
-        num_qubits = max(max(coupling_map)) + 1 if coupling_map is not None else 1
-
-        self._qubits = list(range(num_qubits))
-        self._coupling_map = coupling_map
+        self._coupling_map = coupling_map or []
```

Comment:
It feels like the coupling map should be a required argument with the way we are using. At least `update_inst_map()` on default parameters won't work properly without it.

I wonder about if requiring the coupling map could be avoided by looking at the qubits referenced in `add_schedule()` and `add_parameter_value()` calls and updating the operated qubits as necessary, but I guess there is still the case of default values that could work any number of sets of qubits until you specify which sets are needed.

Reply:
I agree. This is complicated by single-qubit devices, an edge case. E.g. `Armonk` does not have a coupling map
```
FakeArmonk().configuration().coupling_map
```
will give you None. The line
```
self._coupling_map = coupling_map or []
```
sort of deals with that. To make is more obvious that we require a coupling map I changed it here: [56a9b70](https://github.com/Qiskit/qiskit-experiments/pull/740/commits/56a9b70080156bcf147227a3df0489eba9fbdc98)